src/ECharts/Types.purs

```diff
@@ -83,6 +83,11 @@ radiusToForeign ∷ Radius → Foreign
 radiusToForeign (Radius {start, end}) =
   toForeign [ pixelOrPercentToForeign start, pixelOrPercentToForeign end ]
 
+newtype SingleValueRadius = SingleValueRadius PixelOrPercent
+singleValueRadiusToForeign ∷ SingleValueRadius → Foreign
+singleValueRadiusToForeign (SingleValueRadius r) =
+  toForeign $ pixelOrPercentToForeign r

```

Comment:
I think the `toForeign` is redunant here, since `pixelOrPercentToForeign` already returns `Foreign`.